src/main/java/com/khomishchak/ws/services/exchangers/ExchangerService.java

```diff
@@ -20,6 +21,8 @@ public interface ExchangerService {
 
     List<Balance> getAllMainBalances(long userId);
 
+    double getDepositValueForPeriod(long userId, String ticker, LocalDateTime startingDate, LocalDateTime endingDate);
```

Comment:
Adjust to above. You can use Criteria API(Specification) or make your own for filtering.